net/wireguard/files/wireguard.sh

```diff
@@ -161,6 +140,13 @@ proto_wireguard_setup() {
     exit 1
   fi
 
+  # endpoint dependency
```

Comment:
Declare `wg_endpoint` at the top as a local variable:

```
local wg_endpoint
```

But in light of the next comment, you can just get rid of `wg_endpoint` all together.